lib/src/main/java/com/basistheory/android/view/mask/MaskResult.kt

```diff
@@ -0,0 +1,18 @@
+package com.basistheory.android.view.mask
+
+import android.text.Editable
+
+internal data class MaskResult(
+    val maskedValue: String,
+    val unMaskedValue: String,
+    val isDone: Boolean,
```

Comment:
Should we call this `isComplete` to match the `complete` property in the change event? (we should rename that one to `isComplete` too - it was copied from web [here](https://docs.basistheory.com/elements/#element-events-on-change).

Reply:
Updated accordingly